mccue/mccue-sanjose.md

```diff
@@ -0,0 +1,5 @@
+# San Jose
+
+### Restaurants
+
+### Things to do
```

Comment:
The Earthquakes is the local football (soccer) team. If you like BB, you may have watched the Warriors :unamused: